### DISCUSSION TITLE
Settings Reset

### DISCUSSION THREAD

**Discussion by forgoton74** [winglessraven/AMP-Discord-Bot#119] [General]
Hello, 
After I stop then restart the instance it resets the settings I have selected did i do something wrong when installing the script.

**Comment by winglessraven** [comment]
That's not something I've seen before.

Can you do the following to make sure it's writing the settings correctly...

1) Start your instance
2) Change some settings
3) Check that the `DiscordBotPlugin.kvp` file inside the root instance folder contains the settings you changed

**Reply by forgoton74** [reply]
Sorry I didn't reply back right away I couldn't get it to do it again so i forgot about the problem just letting you know It must have been something i did and didn't realize. Everything works fine now.
